include/boost/url/detail/charsets.hpp

```diff
@@ -0,0 +1,112 @@
+//
+// Copyright (c) 2022 alandefreitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CPPAlliance/url
+//
+
+#ifndef BOOST_URL_DETAIL_CHARSETS_HPP
+#define BOOST_URL_DETAIL_CHARSETS_HPP
```

Comment:
sorry... I meant that this file should go in `<boost/url/rfc/detail/charsets.hpp>`